### PR TITLE
Add more hints of project2b

### PR DESCRIPTION
**Pull request by h3n4l**
Failure to handle the proposal logic correctly will lead to a lot of confusing errors

### CODE REVIEW

File: doc/project2-RaftKV.md [hunk 1]
```diff
@@ -202,6 +202,8 @@ In this stage, you may consider these errors, and others will be processed in pr
 > - Record the callback of the command when proposing, and return the callback after applying.
 > - For the snap command response, should set badger Txn to callback explicitly.
 > - After 2A, some tests you may need to run them multiple times to find bugs
+> - Use the log package to help debug code that you think may be problematic
+> - Handle `proposal` logic carefully
```
Comment (Connor1996) [inline doc/project2-RaftKV.md:206]: it's meaningless, every logic should be handled carefully

File: doc/project2-RaftKV.md [hunk 2]
```diff
@@ -202,6 +202,8 @@ In this stage, you may consider these errors, and others will be processed in pr
 > - Record the callback of the command when proposing, and return the callback after applying.
 > - For the snap command response, should set badger Txn to callback explicitly.
 > - After 2A, some tests you may need to run them multiple times to find bugs
+> - Use the log package to help debug code that you think may be problematic
```
Comment (Connor1996) [inline doc/project2-RaftKV.md:205]: That's what the hint `Set the environment variable LOG_LEVEL=debug which may help you in debugging, see also the all available log levels.` in Part B indicates

Comment (h3n4l) [reply doc/project2-RaftKV.md:205]: Thanks

Comment (h3n4l) [reply doc/project2-RaftKV.md:205]: Thanks your review!

**Pull request closed without merging**